deepchem/feat/molecule_featurizers/__init__.py

```diff
@@ -9,6 +9,7 @@
 from deepchem.feat.molecule_featurizers.mordred_descriptors import MordredDescriptors
 from deepchem.feat.molecule_featurizers.mol2vec_fingerprint import Mol2VecFingerprint
 from deepchem.feat.molecule_featurizers.one_hot_featurizer import OneHotFeaturizer
+from deepchem.feat.molecule_featurizers.sparse_matrix_one_hot_featurizer import SparseMatrixOneHotFeaturizer
```

Comment:
I think we should add a new folder `featurizers/sequence_featurizers` since we are starting to get more sequence featurizations that are not directly relevant to molecules. I think this can be done in a separate PR since we may want to consider moving `OneHotFeaturizer` and `SparseMatrixOneHotFeaturizer` there in addition to future multiple sequence alignment featurizations.